packages/edit-site/src/components/global-styles/shadows-edit-panel.js

```diff
@@ -381,8 +379,7 @@ function ShadowItem( { shadow, onChange, canRemove, onRemove } ) {
 					<HStack align="center" justify="flex-start" spacing={ 0 }>
 						<FlexItem style={ { flexGrow: 1 } }>
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> shadows --> add shadow. Open shadow and check for `drop shadow` button.

**Note:** It already has a 40px size as height is auto.

| Before | After     |
| :-------- | :------- |
| <img width="625" alt="drop-showdow-button-before" src="https://github.com/user-attachments/assets/c0fb1e82-6165-4107-8ed2-ccd98d345125"> | <img width="625" alt="drop-shadow-button-aafter" src="https://github.com/user-attachments/assets/3f5d7d50-2900-49c6-8480-fb145dac2cb2"> |